ClosedXML/Excel/PivotTables/XLPivotTable.cs

```diff
@@ -80,6 +81,122 @@ public IEnumerable<IXLPivotField> ImplementedFields
 
         public XLPivotTableTheme Theme { get; set; }
 
+        public IXLPivotTable CopyTo(IXLCell targetCell)
+        {
+            var addressComparer = new XLAddressComparer(ignoreFixed: true);            
+            if (addressComparer.Equals(targetCell.Address, this.TargetCell.Address))
+                throw new InvalidOperationException("Cannot copy pivot table to the target cell.");
+
+            var targetSheet = targetCell.Worksheet;
+
+            var pivotTableName = this.Name;
+
+            int i = 0;
+            while (!XLHelper.ValidateName("pivot table", pivotTableName, "", targetSheet.PivotTables.Select(pvt => pvt.Name), out String message))
+            {
+                i++;
+                pivotTableName = this.Name + i.ToInvariantString();
+            }
+
+            XLPivotTable newPivotTable;
+            if (this.SourceType == XLPivotTableSourceType.Table)
+                newPivotTable = targetSheet.PivotTables.Add(pivotTableName, targetCell, this.SourceTable) as XLPivotTable;
+            else if (this.SourceType == XLPivotTableSourceType.Range)
+                newPivotTable = targetSheet.PivotTables.Add(pivotTableName, targetCell, this.SourceRange) as XLPivotTable;
+            else
+                throw new NotImplementedException();
+
+            newPivotTable.RelId = null;
+
+            newPivotTable.Theme = Theme;
+
+            IXLPivotField copyPivotField(IXLPivotField originalPivotField, IXLPivotField newPivotField)
+            {
+                newPivotField
+                    .SetSort(originalPivotField.SortType)
+                    .SetSubtotalCaption(originalPivotField.SubtotalCaption)
+                    .SetIncludeNewItemsInFilter(originalPivotField.IncludeNewItemsInFilter)
+                    .SetRepeatItemLabels(originalPivotField.RepeatItemLabels)
+                    .SetInsertBlankLines(originalPivotField.InsertBlankLines)
+                    .SetShowBlankItems(originalPivotField.ShowBlankItems)
+                    .SetInsertPageBreaks(originalPivotField.InsertPageBreaks)
+                    .SetCollapsed(originalPivotField.Collapsed);
+
+                if (originalPivotField.SubtotalsAtTop.HasValue)
+                    newPivotField.SetSubtotalsAtTop(originalPivotField.SubtotalsAtTop.Value);
+
+                foreach (var v in originalPivotField.SelectedValues)
+                    newPivotField.AddSelectedValue(v);
```

Comment:
Consider defining `AddSelectedValues(IEnumerable<>...)` in pivot field class.

Reply:
Done.